components/checkbox.module.css

```diff
@@ -0,0 +1,22 @@
+.checkbox {
+  position: relative;
+  height: 24px;
+  width: 24px;
+  border: 1px solid #d1d6dc;
```

Comment:
if this isnt in the windi theme maybe u should just go for it and add it to the colors?